articles/introduction-to-editing.mdx

```diff
@@ -0,0 +1,138 @@
+
+1. `cloudcannon.config.json`
+2. `cloudcannon.config.yaml`
+3. `cloudcannon.config.yml`
+4. `cloudcannon.config.js`
+5. `cloudcannon.config.cjs`
+
+The global configuration file is the heart of your CloudCannon experience. If you don’t have one, CloudCannon will prompt you to create a global configuration file when you create a new site.
+
+For a more in-depth explanation of the global configuration file, including how to create your configuration file, make a custom file path, or use our configuration GUI, read our documentation:
+
+* [Create a global configuration file](/documentation/articles/create-a-global-configuration-file/)
+
+## What is a Collection?
+
+A Collection is a group of related files with a repeated format stored in the same folder/path within your repository. Collections could be a folder of pages, blog posts, data files, or any other file type. You can edit your Collections by altering any file or adding more files. Collections are the starting point for any new site on CloudCannon.
+
+Let’s cover an example. Your team publishes a weekly blog for your site. Each blog post is a single MDX file stored in the folder `content/posts` within your Git repository. CloudCannon makes creating a new blog post easy, especially for non-technical members of your team who may be hesitant to create new files in a Git repository. In CloudCannon, you can create new collection items with a click of a button, edit them in one of our editing interfaces, and see your existing files in a sortable, filterable browser. All your team’s changes will push back to your Git repository.
+
+For a more in-depth explanation of Collections, including how to display Collections in the site navigation, control who can create new posts, and customize the default content, name, and editing interface of new files, read our documentation:
+
+* [Define your collections](/documentation/articles/define-your-collections/)
+
+## Supported file types
+
+CloudCannon supports the following file formats for standalone data files or Collections:
+
+* Structured data files: JSON, YAML, TOML, CSV, TSV
+* Markup files: HTML, Markdown, MDX
+* Combination files: HTML with front matter, Markdown with front matter, MDX with front matter
+
+Structured data files are a great way to store data. Each key can contain strings, numbers, rich text, or even arrays/objects, which can recurse indefinitely. We recommend structured data files for page-building components, navigation, header and footer, site metadata, or any file that does not contain long-form prose.
+
+Markup files should store more complex data, such as files with multiple lines of text or complex formatting. We recommend markup files for blog posts, articles, or documentation.
```

Comment:
I don't know if "more complex data" is the right phrasing here.

I would say something like:

```suggestion
Markup files should store loosely structured content, such as files with multiple lines of text or complex formatting. We recommend markup files for blog posts, articles, or documentation.
```

```suggestion
Markup files should store longer written content, such as files with multiple lines of text or complex formatting. We recommend markup files for blog posts, articles, or documentation.
```

```suggestion
Markup files should store free-form content, such as files with multiple lines of text or complex formatting. We recommend markup files for blog posts, articles, or documentation.
```